tests/main/confinement-classic/task.yaml

```diff
@@ -17,6 +17,8 @@ execute: |
     run_install --classic
     $SNAPMOUNTDIR/bin/test-snapd-hello-classic | MATCH 'Hello Classic!'
 
+    TMPDIR=/tmpdir $SNAPMOUNTDIR/bin/test-snapd-hello-classic t | MATCH TMPDIR=/tmpdir
```

Comment:
This fails with "/bin/bash: line 65: SNAPMOUNTDIR: unbound variable" which doesn't make any sense to me. Also the instructions in HACKING.md for running the spread tests don't work. Help?

Reply:
If you have the a key for linode API access (aka spread key) you can run this test with:

`spread -debug -v linode:ubuntu-16.04-64:tests/main/confinement-classic`

(substitute target environment if you want other OS)

I would also suggest to cut on the number of workers to just one (see spread.yaml) as otherwise debugging is weird and you waste machines that do nothing useful.